ceph/rbd/mirror/bootstrap.py

```diff
@@ -0,0 +1,62 @@
+"""Module to hold methods of rbd mirror pool peer bootstrap
+
+This module has all subcommands of the command -
+rbd mirror pool peer bootsrap
+"""
+
+from ceph.rbd.mirror.peer import Peer
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class Bootstrap(Peer):
+    def __init__(self, nodes):
+        self.nodes = nodes
+        super(Bootstrap, self).__init__(nodes=nodes)
+
+    def create(self, args):
+        """Wrapper for pool peer bootstrap create.
+
+        Create a peer bootstrap token to import in a remote cluster.
+
+        Args:
+            pool_name: Name of the pool of which peer is to be bootstrapped.
+            site_name: Name of the site for rbd-mirror
+                       name of both sites should be the same
+            file_path: Provide the file name if key needs to be stored in
+                       a file instead of returning.
+
+        Returns:
+            out: Output after execution of command
+                 Out would contain the key.
+            err: Error after execution of command
+        """
+        cmd = f'rbd mirror pool peer bootstrap create {args["pool_name"]}'
+        if args.get("site_name", None):
+            cmd = cmd + f" --site-name {args['site_name']}"
+        if args.get("file_path", None):
```

Comment:
IMO, we can port this write_to_file implementation to an utility which can be reusable. What do you say?

Reply:
Yes, Moved it out of sdk layer and to utility
Sounds good to stick with only commands in sdk layer